src/token/CountdownERC721.sol

```diff
@@ -265,7 +274,15 @@ contract CountdownERC721 is NonReentrant, ERC721H, ICountdownERC721 {
 
     setStatus(1);
 
-    return _init(initPayload);
+    _init(initPayload);
+
+    (bool success, ) = msg.sender.call(
+      abi.encodeWithSignature("setOwner(address)", initializer.initialOwner)
+    );
+
+    // require(success, "HOLOGRAPH: initOwner failed");
```

Comment:
I think we need to figure out why this isn't returning true for success. even if it still is setting the owner that seems a little concerning

Reply:
Actually working on that 👍🏻

Reply:
The following commit fixed that
https://github.com/holographxyz/holograph-protocol/pull/304/commits/627064390834dcfe4e0aff8d0df8b58003dbf360